stairway/src/main/java/bio/terra/stairway/FlightMap.java

```diff
@@ -55,130 +46,123 @@ public FlightMap() {
    */
   List<FlightInput> makeFlightInputList() {
     ArrayList<FlightInput> inputList = new ArrayList<>();
-    for (Map.Entry<String, Object> entry : map.entrySet()) {
-      try {
-        String value = getObjectMapper().writeValueAsString(entry.getValue());
-        inputList.add(new FlightInput(entry.getKey(), value));
-      } catch (JsonProcessingException ex) {
-        throw new JsonConversionException("Failed to convert value to json string", ex);
-      }
+    for (Map.Entry<String, String> entry : map.entrySet()) {
+      inputList.add(new FlightInput(entry.getKey(), entry.getValue()));
     }
     return inputList;
   }
 
   /**
    * Depending on what version of code a Flight log was written with, we may have a JSON Map, a
    * {@code List<FlightInput>}, both, or neither at deserialization time. This method is used to
-   * generate an {@code Optional<FlightMap>} based on what was contained in the database. This will
-   * return an empty {@code Optional<FlightMap>} if json is null.
+   * generate a FlightMap based on what was contained in the database.
+   *
+   * <p>Flights written with:
+   *
+   * <p>* Version < 0.0.50: Only json is valid, inputList is always empty.
+   *
+   * <p>* Version 0.0.50 - 0.0.60: json and inputList both valid and must be consistent with one
+   * another.
+   *
+   * <p>* Version >= 0.0.61: json always null, inputList is always valid (possibly empty).
```

Comment:
Very minor point; it looks like you're trying to create an unordered list here, if so, it would be be better to use `ul`, `li`.
```suggestion
   * <p>Flights written with:
   *
   * <ul><li>Version < 0.0.50: Only json is valid, inputList is always empty.
   *
   * <li>Version 0.0.50 - 0.0.60: json and inputList both valid and must be consistent with one
   * another.
   *
   * <li>Version >= 0.0.61: json always null, inputList is always valid (possibly empty).
   * </ul>
```

Reply:
Thanks, this was helpful!